doc/introduction/compiling_workflows.rst

```diff
@@ -120,6 +123,47 @@ global state is involved.
 Please refer to the :doc:`AutoGraph guide<catalyst:dev/autograph>` for a
 complete discussion of the supported and unsupported use-cases.
 
+CUDA Quantum
+------------
+
+The PennyLane :func:`.qjit` decorator  can also be used to compile programs
+using `CUDA Quantum <https://pennylane.ai/qml/glossary/what-is-cuda-quantum/>`__,
+a hybrid compiler toolchain by NVIDIA.
+
+First, Catalyst and CUDA Quantum need to be installed:
+
+.. code-block:: bash
+
+    pip install pennylane-catalyst cuda_quantum
+
+Then, simply specify ``compiler="cuda_quantum"`` in the ``@qjit``
+decorator:
+
+.. code-block:: python
+
+    dev = qml.device("softwareq.qpp", wires=2)
+
+    @qml.qjit(compiler="cuda_quantum")
+    @qml.qnode(dev)
+    def circuit(x):
+        qml.RX(x[0], wires=0)
+        qml.RY(x[1], wires=1)
+        qml.CNOT(wires=[0, 1])
+        return qml.expval(qml.PauliY(0))
+
+>>> circuit(jnp.array([0.5, 1.4]))
+-0.47244976756708373
+
+The following devices are available when compiling with CUDA Quantum:
+
+* ``softwareq.qpp``: a modern C++ statevector simulator
+* ``nvidia.custatevec``: The NVIDIA CuStateVec GPU simulator (with support for multi-gpu)
+* ``nvidia.cutensornet``: The NVIDIA CuTensorNet GPU simulator (with support for matrix product state)
```

Comment:
What would be the motivation for `softwareq.qpp`? Just to check things are working on CPU before going to GPU?

(not a blocker for this PR)

Reply:
Yep! The other two don't support CPUs